api/src/main/java/com/datastrato/gravitino/rel/Column.java

```diff
@@ -23,6 +24,7 @@
 public interface Column {
 
   Expression DEFAULT_VALUE_NOT_SET = () -> Expression.EMPTY_EXPRESSION;
+  Expression DEFAULT_VALUE_OF_CURRENT_TIMESTAMP = FunctionExpression.of("current_timestamp");
```

Comment:
it's a little odd to place `DEFAULT_VALUE_OF_CURRENT_TIMESTAMP` in column API

Reply:
@jerryshao What's your sugsestion?

Reply:
@FANNG1 what is your suggestion?

Reply:
I'm OK to put it here. We don't have to create a new interface for these two variables. If later on there're more variables needs to manage, we can deprecate this and point to the new place.